src/hipscat/pixel_math/README.md

```diff
@@ -154,6 +154,75 @@ faces).
 #### Implementation
 Pretty straightforward implementation of the algorithm above.
 
+### pixel_is_polar
+Because of the nature of spherical coordinate systems, hipscat runs into some tricky edge cases at the poles. to ensure we can appropriately handle those problems, we need to check if a pixel is one of the four 'polar pixels'.
+
+#### Algorithm
+In the ring id scheme for `healpix`, the first and last 4 pixels are the polar pixels. To determine whether a nest scheme pixel is at the poles, all we have to do is convert the pixel into the ring scheme and determine if it falls at the beginning or end of the range 0 -> npix. So, if in the ring scheme the pix is `<= 3`, or `>= npix - 4`, we can safely assume that it is a polar pixel.
+
+### get_truncated_margin_pixels
+For pixels that are at the poles, our margin bounding box isn't set up to handle the data that is on the other side of the hemisphere from the partition. So when we calculate the boundaries, we truncate declination values outside of the range -90 -> 90. However, we obviously still want to include neighbor margin data that is affected by this truncation, namely the 3 margin pixels that are also polar pixels.
+
+#### Algorithm
+We want to find the 3 pixels at the healpix order of our margins that are polar. This is all of the pixels around the given pole _except_ the one that is contained within our partition pixel. 
+
+For the north pole, this is straightforwardly done by converting our partition pixel into the ring id scheme and returning the values between 0 and 3 that aren't equal to it.
+
+For the south pole, we have to do a little more complicated math, due to the fact that the southern polar pixels aren't the same values at any healpix order. To find the at polar pixel at the `margin_order` that is contained by the partition pixel (and therefore to be excluded), we can take advantage of the fact that in the nest scheme the southern partition of a pixel is equal to `4 * pix` (see the algorithm section of `get_edge` above for more info), so to get the excluded southern pixel all we have to do is find the difference between `order` and `margin_order`, and multiply our `pix` value by `4 ** d_order`.
```

Comment:
I think it's consistent. The complication is if the margin pixels and the partition pixel are at different orders, you need to do the conversion of the polar partition pixel number to the margin pixel order so you can exclude the right pixel at the margin order, then test the others that are >= npix-4. At least I think that's how it works.

Reply:
yes exactly, that's how it works. in a the ring scheme the north polar pixels are always 0 -> 3, and polar pixel `i` will contain the polar pixel `i` at a higher order, so all we have to do to find the exclude pixel is convert the pixel from nest into the ring scheme.

with the south pole, it's not so simple. when comparing between `order` and `margin_order`, the npix grows so we can't just convert the pixel into ring to find the excluded polar pixel. so the process that we have to do is find the southern most `margin_order` pixel contained by the initial polar partition pixel (which is just `4 ** (margin_order - order) * pix`, because the nest scheme is really clever :) ), and then convert that pixel into the ring scheme.

Reply:
Got it. Thanks.